tests/testthat/test-target_market_share.R

```diff
@@ -1175,3 +1177,135 @@ test_that("`target_market_share` outputs as expected for companies with 0
   expect_equal(out$projected$technology_share, c(0, 1))
   expect_equal(out$target_sds$technology_share, c(0, 0))
 })
+
+test_that("outputs columns `percent_change_by_scope` and `scope`", {
+  out <- target_market_share(
+    fake_matched(),
+    fake_ald(),
+    fake_scenario(),
+    region_isos_stable
+  )
+
+  expected_added_columns <- c( "scope", "percentage_of_initial_production_by_scope")
+
+  expect_equal(setdiff(expected_added_columns, names(out)),character(0))
+})
+
+test_that("with known input outputs `percent_of_initial_production_by_scope` as
+          expected, at portfolio level", {
+            portfolio <- fake_matched(
+              name_ald = c("comp1", "comp2")
+            )
+
+            ald <- fake_ald(
+              technology = c("electric", "ice", "electric", "ice", "electric", "ice", "electric", "ice"),
+              year = c(2020, 2020, 2021, 2021, 2020, 2020, 2021, 2021),
```

Comment:
generally I believe it's preferred to use integers with the `2020L` syntax when relevant